lib/Resolver.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Trails Resolver Class.
+ */
+module.exports = class TrailsResolver {
```

Comment:
is there any example of model/resolver somewhere? See how it's look like and how it's setup

Reply:
I wrote some docs on Resolvers here: https://trailsjs.io/doc/en/build/resolver